src/vivarium/framework/population.py

```diff
@@ -4,10 +4,10 @@
 ================================
 
```

Comment:
Open a docstring only PR for everyone to review now. This is much easier than the concept stuff.